rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -458,10 +464,19 @@ public NodesRecoveryManager apply(RMCore rmCore) {
         };
     }
 
-    @VisibleForTesting
-    boolean restoreNodesAndNodeSources() {
+    private void recoverNodeSourcesAndNodes() {
         Collection<NodeSourceData> nodeSources = dbManager.getNodeSources();
 
+        if (nodeSources.isEmpty()) {
```

Comment:
can you move this into a separate method called longInfo(nodeSources) or something like that?

Reply:
applied